### ISSUE TITLE
Plain Web project with Spring Boot 2.0.0.RC1 fails to start

### ISSUE THREAD

**Issue by gourahari** [spring-projects/spring-boot#11996]
Started a simple spring boot project (spring-boot: 2.0.0.RC1) with just "**web**" dependency and upon running the app, getting the following error.

```
Description:

The Bean Validation API is on the classpath but no implementation could be found

Action:

Add an implementation, such as Hibernate Validator, to the classpath

```

pom.xml:

```
    <parent>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter-parent</artifactId>
        <version>2.0.0.RC1</version>
    </parent>

	<dependencies>
		<dependency>
			<groupId>org.springframework.boot</groupId>
			<artifactId>spring-boot-starter-web</artifactId>
		</dependency>

		<dependency>
			<groupId>org.springframework.boot</groupId>
			<artifactId>spring-boot-starter-test</artifactId>
			<scope>test</scope>
		</dependency>
	</dependencies>
```

**Comment by wilkinsona** [comment]
It being on the classpath will not make any difference if the jar file is corrupted. I'd delete it from your Maven cache and try building your app again.

**Comment by gourahari** [comment]
I agree @wilkinsona. As I mentioned in my earlier comment, "I even tried removing the whole .m2 repo before posting the issue here.". But no luck.
However, a project created from spring starter couple of weeks ago with spring boot "2.0.0.M7" and it was okay.
Even it runs okay now as well.

**Comment by wilkinsona** [comment]
Spring Boot RC1 uses Hibernate Validator 6.0.7 while Spring Boot 2.0.0M7 uses Hibernate Validator 6.0.5. I would guess that you aren't experiencing a problem with downloading Hibernate Validator 6.0.5 but you are experiencing a problem with Hibernate Validator 6.0.7. Perhaps you're being a Maven proxy that has cached a corrupted copy of 6.0.7, for example?

If you need further help with this, please chat on Gitter or ask a question on Stack Overflow. I haven't seen anything to suggest that there's a problem with Spring Boot RC1 (its build would have failed if Hibernate Validator integration didn't work) and, unfortunately, we don't have time to help users diagnose general Maven and networking problems.

**Comment by gourahari** [comment]
Thanks
